a10_octavia/tests/unit/controller/worker/tasks/test_network_tasks.py

```diff
@@ -0,0 +1,210 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+
+import copy
+import imp
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from octavia.common import data_models as o_data_models
+from octavia.network import data_models as o_net_data_models
+
+from a10_octavia.common import data_models
+from a10_octavia.controller.worker.tasks import a10_network_tasks
+from a10_octavia.tests.common import a10constants
+from a10_octavia.tests.unit import base
+
+MEMBER = o_data_models.Member()
+VTHUNDER = data_models.VThunder()
+SUBNET = o_net_data_models.Subnet()
+PORT = o_net_data_models.Port()
+
+
+class TestNetworkTasks(base.BaseTaskTestCase):
```

Comment:
Also need a test which covers when no floating ip is specified but a vrrp port id entry exists in the db

Reply:
will add UTs after pair programming session.

Reply:
added UT